main/solution/lambda-website/docker-website/website/placeholder

```diff
@@ -0,0 +1 @@
+## this file is just for website folder not empty
```

Comment:
the community always uses an empty file with name `.keep`

Reply:
Thanks, add it